package/upgrade/Dockerfile

```diff
@@ -1,3 +1,5 @@
+FROM registry.opensuse.org/isv/rancher/harvester/toolbox/main/elemental-toolbox:latest AS temp_elemental
```

Comment:
any concerns about the lint error @Vicente-Cheng, are we possible to use a minor tag version e,g, v1.2?

Reply:
We do not maintain the elemental-toolkit on OBS.
If we want to maintain our elemental-toolkit version with the release, we need to re-originate this package and make it under the OS (e.g., for v1.2, we will create another toolbox image on the Harvester:OS:v1.2:toolbox).

Do we need to handle this release? We might not need to wrap the elemental again once we upgrade to the latest elemental-toolkit. WDUT?

Reply:
Hi @guangbochen, after checking how OBS tags.
I found the elemental team already has the related image, so I changed the image to this.
Please recheck it. Thanks!

Reply:
thanks for the update, can u please help to share the related OBS repo just for reference, thanks.

Reply:
Sure, add the source reference.